applications/nrf5340_audio/src/utils/audio_nrf5340_dk_init.c

```diff
@@ -81,7 +83,7 @@ static int channel_assign_check(void)
 	return 0;
 }
 
-int nrf5340_audio_dk_init(void)
+int audio_nrf5340_dk_init(void)
```

Comment:
Discuss naming. Since this is for both audio and non-audio DK, it may just be named "dk_init"

Reply:
After discussion this file and function are renamed with peripherals.[c,h] and periperals_init().